routes/v1.js

```diff
@@ -81,7 +81,7 @@ module.exports = function(appObj) {
 
     return {
         path: '/',
-        api_version: 1,
```

Comment:
Clearly, the precedent is to use an HTTP resource for the version, so I'm sure I'm wading into a mess by asking, but what about using the `Accept` header?  i.e. `Accept: application/vnd.surge.v1+json`